### DISCUSSION TITLE
Heimdall 2.6.0/2.6.1 update issue

### DISCUSSION THREAD

**Discussion by AlexGeek77** [tteck/Proxmox#2517] [General]
I know the update script says 2.6.0 will break the installation but I decided to try it anyways for fun. Sure enough it broke the installation. The main issue holding up running the service was
```
Feb 19 15:29:06 heimdall php[1066]:   Class "Facade\Ignition\IgnitionServiceProvider" not found
```
For me to fix it I had to install some php tools so I ran `apt install composer` and then I could run `composer dump-autoload` which was able to fix my installation. Now I'm happily running v2.6.1 on my server.

**Comment by tteck** [comment]
Fixed, thanks for your help!
